iOS/Flows/Conversation/Status View/Reactions/AddReactionView.swift

```diff
@@ -14,13 +14,7 @@ class AddReactionView: UICollectionReusableView {
     let imageView = DisplayableImageView()
     let button = Button()
     var taskPool = TaskPool()
-    var message: Message? {
-        didSet {
-            if self.button.menu.isNil {
-                self.button.menu = self.configureMenu()
-            }
```

Comment:
As far as I can see, there's no way to get rid of that UILog when setting the button menu. My solution is to just limit how many times we set it.

See below...